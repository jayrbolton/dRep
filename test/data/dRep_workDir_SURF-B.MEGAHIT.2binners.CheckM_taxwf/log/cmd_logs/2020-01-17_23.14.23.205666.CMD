/miniconda/bin/checkm qa /kb/module/work/tmp/dRep_workDir_d7f80abb-1c52-43db-9fae-de726bb861a2/data/checkM/checkM_outdir/Bacteria.ms /kb/module/work/tmp/dRep_workDir_d7f80abb-1c52-43db-9fae-de726bb861a2/data/checkM/checkM_outdir/ -f /kb/module/work/tmp/dRep_workDir_d7f80abb-1c52-43db-9fae-de726bb861a2/data/checkM/checkM_outdir/Chdb.tsv -t 8 --tab_table -o 2
