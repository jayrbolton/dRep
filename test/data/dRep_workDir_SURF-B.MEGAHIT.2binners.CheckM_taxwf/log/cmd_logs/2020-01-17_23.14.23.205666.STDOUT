[2020-01-17 23:14:23] INFO: CheckM v1.1.1
[2020-01-17 23:14:23] INFO: checkm qa /kb/module/work/tmp/dRep_workDir_d7f80abb-1c52-43db-9fae-de726bb861a2/data/checkM/checkM_outdir/Bacteria.ms /kb/module/work/tmp/dRep_workDir_d7f80abb-1c52-43db-9fae-de726bb861a2/data/checkM/checkM_outdir/ -f /kb/module/work/tmp/dRep_workDir_d7f80abb-1c52-43db-9fae-de726bb861a2/data/checkM/checkM_outdir/Chdb.tsv -t 8 --tab_table -o 2
[2020-01-17 23:14:23] INFO: [CheckM - qa] Tabulating genome statistics.
[2020-01-17 23:14:23] INFO: Calculating AAI between multi-copy marker genes.
[2020-01-17 23:14:23] INFO: Reading HMM info from file.
[2020-01-17 23:14:23] INFO: Parsing HMM hits to marker genes:
[2020-01-17 23:14:25] INFO: QA information written to: /kb/module/work/tmp/dRep_workDir_d7f80abb-1c52-43db-9fae-de726bb861a2/data/checkM/checkM_outdir/Chdb.tsv
[2020-01-17 23:14:25] INFO: { Current stage: 0:00:02.121 || Total: 0:00:02.121 }
